src/test/java/com/upgrad/BankingSystemTest.java

```diff
@@ -55,13 +55,17 @@ public void testWithDrawSuccess(){
     @Test(expected = IllegalArgumentException.class)
     public void testWithDrawInSufficientFunds(){
         double withDrawAmount = 10000;
-        when(accountMock.getBalance()).thenReturn(6000.0);
 
         double updatedAmount=bankingSystem.withdraw(accountNumber,withDrawAmount);
     }
 
 
 
+    public void testWithDrawInOutOFFunds(){
```

Comment:
you have to add the @Test annotation which is missing

Reply:
Test annotation is added